packages/origin-247-transfer/src/issue.service.ts

```diff
@@ -23,13 +22,15 @@ export class IssueService {
     ) {}
 
     public async issueTask() {
-        const etrs = await this.etrRepository.findByState(State.IssuanceAwaiting);
+        const etrs = await this.etrRepository.findByState(
+            State.IssuanceAwaiting,
+            this.batchConfiguration.issueBatchSize
+        );
```

Comment:
I'd put it in options object

```suggestion
        const etrs = await this.etrRepository.findByState(
            State.IssuanceAwaiting,
            { limit: this.batchConfiguration.issueBatchSize }
        );
```

Reply:
Done